src/ILVerification/README.md

```diff
@@ -0,0 +1,3 @@
+# ILVerification
+
+The ILVerification library is part of the ILVerify project. See details under [src/ILVerify/](https://github.com/dotnet/corert/tree/master/src/ILVerify).
```

Comment:
Nit: It would be nice to use relative URLs for the relative tree links so that the links stay valid when the repo is forked or branched.